### PR TITLE
[LWIMIQ] Just doing Fine (2017-05-08)

### PR DESCRIPTION
**Pull request by skateman**

### CODE REVIEW

**Comment by JPrause** [comment]
Thanks @skateman for the submission.
@syncrou @imtayadeway @jntullo can you help review this blog post.

File: site/_posts/2017-05-08-doing-just-fine.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+## Fixed
+Our [charts-guy](https://github.com/PanSpagetka) accidentally deleted a Capacity and Utilization charts under the Availability Zones, he was proven guilty and his punishment was to [put it back where it was](https://github.com/ManageIQ/manageiq/pull/14938). There was also a missing ~person~ [character](https://github.com/ManageIQ/manageiq-ui-classic/pull/1264) in the `CatalogController`, but the [ranger of the wild east](https://github.com/himdel) found it quickly.
+
+## New
+*Úgy gondoljuk, hogy a többnyelvűség támogatása nagyon fontos, ezért folyamatosan dolgozunk ennek fejlesztésén.* This messages was not written in English, because we are constantly working on the internationalization of the project. One step at the time: [Chris](https://github.com/chalettu) created [a set of scripts](https://github.com/ManageIQ/manageiq-ui-service/pull/725) that can to download/upload translation resources from/to our translation server, [Zanata](http://zanata.org/).
```
Comment (JPrause) [inline site/_posts/2017-05-08-doing-just-fine.md]: was this intended to be; "...that **you** can **use** to download/upload translation resources from/to our translation server"

File: site/_posts/2017-05-08-doing-just-fine.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Last Week in ManageIQ: Doing just Fine"
+author: skateman
+date: 2017-05-09
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hi! As you now, we're all doing just *Fine* (pun intended) and I would like to discuss some of the details. So let's get started.
```
Comment (jntullo) [inline site/_posts/2017-05-08-doing-just-fine.md]: should this be "As you know"?

File: site/_posts/2017-05-08-doing-just-fine.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## New
+*Úgy gondoljuk, hogy a többnyelvűség támogatása nagyon fontos, ezért folyamatosan dolgozunk ennek fejlesztésén.* This messages was not written in English, because we are constantly working on the internationalization of the project. One step at the time: [Chris](https://github.com/chalettu) created [a set of scripts](https://github.com/ManageIQ/manageiq-ui-service/pull/725) that can to download/upload translation resources from/to our translation server, [Zanata](http://zanata.org/).
+
+## Deleted
+There was some copy-pasted code on the topology screens that has been refactored and [hundreds of lines of code disappeared](https://github.com/ManageIQ/manageiq-ui-classic/pull/1288) in no time. Also in the topologies, there was a mixin included multiple times instead unnecessarily, [until now](https://github.com/ManageIQ/manageiq-ui-classic/pull/1268).
```
Comment (jntullo) [inline site/_posts/2017-05-08-doing-just-fine.md]: Should this read "there was a mixin included multiple times unnecessarily"

**Comment by jntullo** [comment]
Nice job @skateman 😄

**Comment by JPrause** [comment]
@imtayadeway @syncrou did you have any last comments.

@skateman have you made the suggested edits.

**Comment by skateman** [comment]
@JPrause yes

**Comment by JPrause** [comment]
@cybette please merge at your convenience.

File: site/_posts/2017-05-08-doing-just-fine.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Last Week in ManageIQ: Doing just Fine"
+author: skateman
+date: 2017-05-09
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hi! As you know? We're all doing just *Fine* (pun intended) and I would like to discuss some of the details. So let's get started.
```
Comment (cybette) [inline site/_posts/2017-05-08-doing-just-fine.md]: The question mark after "As you know" seems out of place. Unless there's something you're trying to say that I don't know? :)

File: site/_posts/2017-05-08-doing-just-fine.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+## Fixed
+Our [charts-guy](https://github.com/PanSpagetka) accidentally deleted a Capacity and Utilization charts under the Availability Zones, he was proven guilty and his punishment was to [put it back where it was](https://github.com/ManageIQ/manageiq/pull/14938). There was also a missing ~person~ [character](https://github.com/ManageIQ/manageiq-ui-classic/pull/1264) in the `CatalogController`, but the [ranger of the wild east](https://github.com/himdel) found it quickly.
+
+## New
+*Úgy gondoljuk, hogy a többnyelvűség támogatása nagyon fontos, ezért folyamatosan dolgozunk ennek fejlesztésén.* This messages was not written in English, because we are constantly working on the internationalization of the project. One step at the time: [Chris](https://github.com/chalettu) created [a set of scripts](https://github.com/ManageIQ/manageiq-ui-service/pull/725) that you can use to download/upload translation resources from/to our translation server, [Zanata](http://zanata.org/).
```
Comment (cybette) [inline site/_posts/2017-05-08-doing-just-fine.md]: This is cool! Can we get a link to the [ManageIQ project on Zanata](https://translate.zanata.org/project/view/manageiq/) somewhere?

File: site/_posts/2017-05-08-doing-just-fine.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+## Deleted
+There was some copy-pasted code on the topology screens that has been refactored and [hundreds of lines of code disappeared](https://github.com/ManageIQ/manageiq-ui-classic/pull/1288) in no time. Also in the topologies, there was a mixin included multiple times unnecessarily, [until now](https://github.com/ManageIQ/manageiq-ui-classic/pull/1268).
+
+## Wrapping up
+That's all for me, but if you think it's not enough feel free to check out the [list][manageiq PRs merged] [of][manageiq-ui-classic PRs merged] [all][manageiq-ui-service PRs merged] [our][manageiq-providers-amazon PRs merged] [merged][manageiq-providers-azure PRs merged] [PRs][manageiq-providers-vmware PRs merged].
```
Comment (cybette) [inline site/_posts/2017-05-08-doing-just-fine.md]: Suggestion: That's all **from** me

(as in, that's all from the author for the readers)

**Comment by cybette** [comment]
Sorry for the late comments, great post once again @skateman!

File: site/_posts/2017-05-08-doing-just-fine.md (hunk 4 above)
Comment (skateman) [inline site/_posts/2017-05-08-doing-just-fine.md]: @jntullo told me to use the question mark here, so who should I listen to? :wink:

Comment (cybette) [inline site/_posts/2017-05-08-doing-just-fine.md]: I think @jntullo is asking `should this be "As you know"?` 

i.e. replacing `now` with `know` , but not to add the question mark which is outside of the quotes

File: site/_posts/2017-05-08-doing-just-fine.md (hunk 6 above)
Comment (skateman) [inline site/_posts/2017-05-08-doing-just-fine.md]: Well, I was inspired by @PanSpagetka's finishing line from the daily standups :wink: but will change.

Comment (cybette) [inline site/_posts/2017-05-08-doing-just-fine.md]: ah, I should attend daily standups more often :P This is a minor one, up to you whether you want to change it!

Comment (skateman) [inline site/_posts/2017-05-08-doing-just-fine.md]: Sorry @PanSpagetka :cry:

**Pull request merged**